modules/cli/src/main/kotlin/datamaintain/cli/App.kt

```diff
@@ -40,6 +42,19 @@ class App : CliktCommand() {
 
     private val mongoTmpPath: String? by option(help = "mongo tmp file path")
 
+    private val tags: Set<Tag>? by option(help = "list of your tags defined using glob path matchers, like this: " +
+            "MYTAG1=[pathMatcher1; pathMatcher2],MYTAG2=[pathMatcher3]...")
+            .convert {
+                it.split(",")
```

Comment:
All this block is duplicated with DatamaintainConfig. Can't we find a way to have this code in one place ?